2 - Triplet sampling (Random).ipynb

```diff
@@ -202,22 +210,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def random_triplet_sampling(samples_per_user, hashes_container, desc=None):\n",
+    "def random_triplet_sampling(samples_per_user, hashes_container, desc=None, limit_iteration=10000):\n",
```

Comment:
Why 10000? Maybe we could use the number of interaction as limit, or a proportion of said number. If I have a million records, and need to sample an important number of it, a proportion of `len(interactions_df)` (or `interactions_df.size`, not sure which one is better) would be more appropriate than a fixed number